packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedFieldMessageEmbed.ts

```diff
@@ -0,0 +1,155 @@
+import { MessageEmbed, MessageEmbedOptions } from 'discord.js';
+
+import { PaginatedMessage } from './PaginatedMessage';
+
+/**
+ * This is a utility of {@link PaginatedMessage}, but paginating only the field of an embed.
+ * You must either use this class directly or extend it.
+ *
+ * @example
+ * ```typescript
+ * const { PaginationFieldMessageEmbed } = require('@sapphire/discord.js-utilities');
+ *
+ * new PaginationFieldManager()
+ *    .setTitleField('')
+ *    .setTemplate({ ...options })
+ *    .setItems([{ field: 'value' }, { field: 'value' }, { field: 'value' }, { field: 'value' }])
+ *    .setItemsPerPage(2)
+ *    .make()
+ *    .run(message.author, message.channel);
+ * ```
+ */
+export class PaginationFieldMessageEmbed<Element> extends PaginatedMessage {
+	private embedTemplate: MessageEmbed = new MessageEmbed();
+	private totalPages = 0;
+	private items: Element[] = [];
+	private itemsPerPage = 10;
+	private fieldTitle!: string;
+
+	/**
+	 * Items
+	 *
+	 * Array of items for pagination.
+	 * @param items The pages to set
+	 */
+	public setItems(items: Element[]) {
+		this.items = items;
+		return this;
+	}
+
+	/**
+	 * Title
+	 *
+	 * Title of the embed field that will be used to paginate the items.
+	 * @param title Field title
```

Comment:
```suggestion
	 * Set the title of the embed field that will be used to paginate the items.
	 * @param title Field title
```

Also describe where this title ends up in the embed exactly. I don't understand what this does at all at the moment